force-app/main/default/lwc/stoMessagingContainer/stoMessagingContainer.js-meta.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>59.0</apiVersion>
+    <isExposed>true</isExposed>
+    <masterLabel>STO Messaging Container v. 2</masterLabel>
+    <targets>
+        <target>lightning__RecordPage</target>
+    </targets>
+    <targetConfigs>
+        <targetConfig targets="lightning__RecordPage">
+            <property name="cardTitle" type="String" label="Header title" default="Skriv til oss"></property>
```

Comment:
Kan denne propertyen slettes?

Reply:
Vi trenger denne api variabelen siden komponenten brukes både for thread og case (vi har forskjellige labels for de)